src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -226,11 +241,13 @@ public String getActionComment() {
 
   public Result<Void> isValidStatus() {
     if (!representation.containsKey(STATUS)) {
+      log.warn("isValidStatus:: Loan does not have a status");
       return failedDueToServerError("Loan does not have a status");
     }
 
     // Provided status name is not present in the enum
     if (getStatus() == null) {
+      log.warn("isValidStatus:: Loan status must be \"Open\" or \"Closed\"");
```

Comment:
Can we extract message into a variable and reuse it?

Reply:
done